TypeCobol.Analysis/Cfg/DefaultControlFlowGraphBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace TypeCobol.Analysis.Cfg
+{
+    /// <summary>
+    /// Default Control Flow Graph Builder with any object has Data.
+    /// </summary>
+    public class DefaultControlFlowGraphBuilder : ControlFlowGraphBuilder<object>
```

Comment:
Should be typed otherwise the Data type can not be used. Fixed in DFA.

Reply:
Yes, but by default to instantiate the TypeVariable  (D) to something object is used for CFG as CFG does not use by default custom DATA, DFA uses its own data.